src/modules/card.js

```diff
@@ -0,0 +1,18 @@
+export default function createCard(meal) {
+  const card = document.createElement('div');
+  card.classList.add('col-lg-4');
+  const cardContent = `
+  <div class="card" style="background: linear-gradient(to bottom right, #556B2F, #AFEEEE);">
+    <img src="${meal.strMealThumb}" class="card-img-top img-fluid" alt="Food Image">
+    <div class="card-footer d-flex justify-content-between align-items-center">
+      <small class="text-white">${meal.strMeal}</small>
+      <button type="button" class="btn btn-outline-light btn-icon btn-open-modal">
```

Comment:
[ Optional ]

The card's UI is complete and nicely structured, but I am noticing the wireframe had a comment at the footer too.

![image](https://github.com/rmiyoyo/Spaceship-Kanbad-Board/assets/99050177/d2e3085d-eccd-4a3d-b833-55e7132c57b9)


_PS: I will add it nicely the next feature I must add stems from the comment_